kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/spec/sorters.kt

```diff
@@ -22,7 +22,7 @@ interface SpecSorter {
 /**
  * An implementation of [SpecExecutionOrder] which will order specs in lexicographic order.
  */
-object LexicographicSpecSorter : SpecSorter {
+val LexicographicSpecSorter = object : SpecSorter {
```

Comment:
Breaking change i reckon. Do we want to make it `internal` while we're at it?

Reply:
Sure, makes sense.